src/components/DialogBox/CombinationalAnalysis.vue

```
@@ -200,78 +199,83 @@ function createLogicTable() {
         alertType.value = 'warning'
         alertMessage.value =
             'Use Either Combinational Analysis Or Boolean Function To Generate Circuit!'
-        setTimeout(() => {
+        watch(showAlert, () => {
             showAlert.value = false
-        }, 2000)
+        }, { timeout: 2000 })
     }
 }
 
 function createBooleanPrompt(inputList, outputList, scope = globalScope) {
```

ESLint

[error] 'scope' is assigned a value but never used.

(no-unused-vars)